tests/test_carbonate_sync.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import carbonate_sync
+
+
+# Super basic tests, mostly to check the syntax.
+# TODO(c.chary): Implement more tests.
+class TestCarbonateSync(unittest.TestCase):
```

Comment:
this is effectively supposed to test carbonate_sync.py even if it currently does a very poor job at it. (at least it checks the syntax ..)